cico_release.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# Output command before executing
+set -x
+
+# Exit on error
+set -e
+
+SCRIPT_DIR=$(cd "$(dirname "$0")"; pwd)
+export SCRIPT_DIR
+
+# Source environment variables of the jenkins slave
+# that might interest this worker.
+function load_jenkins_vars() {
+  if [ -e "jenkins-env.json" ]; then
+    eval "$(./env-toolkit load -f jenkins-env.json \
+            DEVSHIFT_TAG_LEN \
+            QUAY_ECLIPSE_CHE_USERNAME \
+            QUAY_ECLIPSE_CHE_PASSWORD \
+            JENKINS_URL \
+            GIT_BRANCH \
+            GIT_COMMIT \
+            BUILD_NUMBER \
+            ghprbSourceBranch \
+            ghprbActualCommit \
+            BUILD_URL \
+            ghprbPullId)"
+  fi
+}
+
+function install_deps() {
+  # We need to disable selinux for now, XXX
+  /usr/sbin/setenforce 0  || true
+
+  # Get all the deps in
+  yum install -y yum-utils device-mapper-persistent-data lvm2
+  yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+  yum install -y docker-ce \
+    git
+
+  service docker start
+  echo 'CICO: Dependencies installed'
+}
+
+function set_release_tag() {
+  # Let's obtain the tag based on the 
+  # version defined in the 'VERSION' file
+  TAG=$(head -n 1 VERSION)
+  export TAG
+}
+
+function tag_push() {
+  local TARGET=$1
+  docker tag "${IMAGE}" "$TARGET"
+  docker push "$TARGET"
+}
+
+function release() {
+  TARGET=${TARGET:-"centos"}
+  REGISTRY="quay.io"
+
+  if [ "$TARGET" == "rhel" ]; then
```

Comment:
this approach will not work for release since the script is running only one time and there is no `target` in place. I believe you need to build & push `centos` first, than login with another creds and push to `rhel`

Reply:
actually, it looks like `extra_target: rhel` in the job definition could do the trick, so the comment above is probably not relevant